appinventor/components/src/com/google/appinventor/components/runtime/Switch.java

```diff
@@ -94,7 +94,7 @@ private ColorStateList createSwitchColors(int active_color, int inactive_color)
    *
    * @return  thumb RGB color with alpha
    */
-  @SimpleProperty(category = PropertyCategory.APPEARANCE, userVisible = false)
+  @SimpleProperty(category = PropertyCategory.APPEARANCE, userVisible = true)
```

Comment:
userVisible = true is the default, so you could just remove the setting of the field here.

Reply:
Done